src/address.rs

```diff
@@ -206,18 +222,19 @@ fn decode() {
 
 #[test]
 fn serialize_null_address() {
-    use serde_rlp::ser::to_bytes;
     let address = Address::new();
-    assert_eq!(to_bytes(&address).unwrap(), [128]);
+    let s = serde_json::to_string(&address).unwrap();
+    assert_eq!(s, r#""0x0000000000000000000000000000000000000000""#);
+    let recovered_addr: Address = serde_json::from_str(&s).unwrap();
+    assert_eq!(address, recovered_addr);
 }
 
 #[test]
 fn serialize_padded_address() {
-    use serde_rlp::ser::to_bytes;
     let address: Address = "00000000000000000000000000000000000000c0".parse().unwrap();
     assert_eq!(
-        to_bytes(&address).unwrap(),
-        [148, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0xc0]
+        serde_json::to_string(&address).unwrap(),
+        r#""0x00000000000000000000000000000000000000c0""#
```

Comment:
If for any reason someone will need to change this test they'll have to alter `"00000000000000000000000000000000000000c0"` twice. Why can't it be stored in a variable?

Reply:
Good catch. Fixed